### ISSUE TITLE
model pretrained weights with no base classes

### ISSUE THREAD

**Issue by ahtchow** [djamahl99/findnpropagate#8]
Hello are you able share the weights for the no base classes, so 10 unknown classes on NuScenes?

**Comment by djamahl99** [comment]
Hi, we did not test this in this work. In the paper the no base classes tests were done with the seed labels (i.e. greedy box seeker proposals) without training. You can adapt the configuration from the other settings to trial this for your own work.

Let me know if you come into any further issues.
